picard/ui/ui_infodialog.py

```diff
@@ -16,7 +16,6 @@ def setupUi(self, InfoDialog):
         self.info_tab = QtWidgets.QWidget()
         self.info_tab.setObjectName("info_tab")
         self.vboxlayout = QtWidgets.QVBoxLayout(self.info_tab)
-        self.vboxlayout.setContentsMargins(0, 0, 0, 0)
```

Comment:
I regenerated the UI files from .ui files. Guess that isn't needed anymore and wasn't updated in the repo. Should I remove it from this PR and make another one?